docs/build-with-powerloom/devnet/getting-started.md

```diff
@@ -171,23 +175,22 @@ or
 zsh build.sh
 ```
 
-#### Step 6: Assign your Burner Wallet
+#### Step 6: Assign your Wallet
 
 ```bash
-Do you want to assign a burner wallet to a slot? (yes/no):
+Do you want to assign a wallet to a slot? (yes/no):
 ```
 
-Enter yes and then enter your burner wallet address when prompted.
-- This is the `SIGNER_ACCOUNT_ADDRESS` that was set during [Step 3](#step-3-configure-the-environment-variables).
+Enter yes and then enter your wallet address when prompted.
```

Comment:
@iamvijaysaran, we never ask for any wallet address; you've run the node multiple times.